Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM nvidia/cuda:9.0-cudnn7-devel
+
+SHELL ["/bin/bash", "-c"]
+
+RUN mkdir /input
+RUN mkdir /output
+RUN mkdir /license
+
+RUN apt-get update && apt-get install -y wget && apt-get install -y git
+RUN MINICONDA="Miniconda3-latest-Linux-x86_64.sh" && \
+    wget --quiet https://repo.continuum.io/miniconda/$MINICONDA && \
+    bash $MINICONDA -b -p /miniconda && \
+    rm -f $MINICONDA
+ENV PATH /miniconda/bin:$PATH
+
+# Add channels and build, then install
+RUN conda config --add channels omnia
+RUN conda update --yes -n base conda
+RUN conda config --add channels conda-forge
```

Comment:
After installing the `conda-forge` channel, you should use `conda update --yes --all`. 
See our recommendation for using `omnia` here:
http://www.omnia.md/install/

Reply:
Ok